### PR TITLE
[#3636] don't set error code

### PR DESCRIPTION
**Pull request by xu-hao**

### CODE REVIEW

File: iRODS/server/core/src/reServerLib.cpp [hunk 1]
```diff
@@ -807,9 +807,9 @@ runRuleExec( reExecProc_t * reExecProc ) {
                                     reiAndArg->rei->msParamArray,
                                     reiAndArg->rei, NO_SAVE_REI );
 
-    if ( reiAndArg->rei->status < 0 ) {
-        reExecProc->status = reiAndArg->rei->status;
-    }
+//    if ( reiAndArg->rei->status < 0 ) {
```
Comment (trel) [inline iRODS/server/core/src/reServerLib.cpp]: Why comment these out rather than just delete?  May I delete?

**Comment by xu-hao** [comment]
Yes, just delete.

**Comment by xu-hao** [comment]
I was trying to figure out if that was the cause of the bug. So I didn't
delete at first, but forgot to think about delete afterwards.

**Comment by xu-hao** [comment]
I can send you a new PR if necessary.

**Comment by trel** [comment]
No, I'm good - it's running in CI.

**Comment by trel** [comment]
I tested the GitHub integrated merge conflict resolution interface - it didn't work quite as I expected.  I've merged from the command line and will just close this directly - sorry no purple this time unless I can figure it out.

**Pull request closed without merging**